tools/GitHubTools.psm1

```diff
@@ -298,9 +298,10 @@ function Copy-GitRepository
     }
 
     Push-Location $Destination
+    $autocrlfSetting = git config core.autocrlf
     try
     {
-        Exec { git config core.autocrlf true }
+        Exec { git config core.autocrlf input }
```

Comment:
What does this do differently?

Reply:
`input` pushes CRLF as LF, `true` does that and pulls LF as CRLF, which is irrelevant for our usage